packages/client/package.json

```diff
@@ -83,6 +83,30 @@
       "import": "./wasm.js",
       "default": "./wasm.js"
     },
+    "./runtime/library": {
+      "types": "./runtime/library.d.ts",
+      "require": "./runtime/library.js",
+      "import": "./runtime/library.js",
+      "default": "./runtime/library.js"
+    },
+    "./runtime/binary": {
+      "types": "./runtime/binary.d.ts",
+      "require": "./runtime/binary.js",
+      "import": "./runtime/binary.js",
+      "default": "./runtime/binary.js"
+    },
+    "./runtime/edge": {
+      "types": "./runtime/edge.d.ts",
+      "require": "./runtime/edge.js",
+      "import": "./runtime/edge.js",
+      "default": "./runtime/edge.js"
+    },
+    "./runtime/wasm": {
+      "types": "./runtime/wasm.d.ts",
+      "require": "./runtime/wasm.js",
+      "import": "./runtime/wasm.js",
+      "default": "./runtime/wasm.js"
+    },
```

Comment:
Allows importing without specifying the extension.

Reply:
I initially had a more compact version (see below), which turned out to be a time sink to make it work with Next.js 13 (misinterpreted in 13 but works in 14), so I opted towards a more verbose and more archaic version of it.

```ts
    "./*.js": "./*.js",
    "./*.d.ts": "./*.d.ts",
    "./*": {
      "types": "./*.d.ts",
      "require": "./*.js",
      "import": "./*.js",
      "default": "./*.js"
    }
```